bindings/wasm/bindings.js

```diff
@@ -190,7 +190,7 @@ Module.setup = function() {
 
   Module.CrossSection.prototype.toPolygons = function() {
     const vec = this._ToPolygons();
-    const result = vec2polygons(vec);
+    const result = vec2polygons(vec, v => [v.x, v.y]);
```

Comment:
Just to double-check, which API do we like better? The one we implemented or the one we documented? Because we could equally update the type instead of this.

Reply:
Personally, since there is a static factory function called `fromPolygons`, I would assume that `toPolygons` would return the same type.